src/lib/modules/Document/DocumentPublicationInfo.js

```diff
@@ -0,0 +1,71 @@
+import { InfoMessage } from '@components/InfoMessage';
+import React, { Component } from 'react';
+import { Divider, Table } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import _isEmpty from 'lodash/isEmpty';
+
+export class DocumentPublicationInfo extends Component {
+  render() {
+    const { publications, documentType } = this.props;
+    if (_isEmpty(publications)) {
+      return (
+        <InfoMessage
+          title="No publication information available."
+          message="This literature has not been published in any journal."
```

Comment:
if it is a book, maybe we don't want to show this? Shall we display it or now based on the `document_type`?

Reply:
From what I remember we decided to display everything, if it is not there to put the information like this one, in order not to have if statements everywhere

Reply:
Agree, but I guess it will be a bit weird in this case? A book "published" in a journal?
Maybe we can put a different message?
```
title="Journal publication"
message="No publication information for this literature"
```
or similar...

Reply:
👍  it sounds better